### ISSUE TITLE
Not accurate calculation of entripic_risk_measure when the value is significantlly big

### ISSUE THREAD

**Issue by masanorihirano** [pfnet-research/pfhedge#580]
```
from pfhedge.nn import EntropicRiskMeasure

loss = EntropicRiskMeasure()
input = -torch.arange(4.0)-100
loss(input)
＃tensor(inf)
```
ref.: https://pfnet-research.github.io/pfhedge/generated/pfhedge.nn.EntropicRiskMeasure.html

This result has to be between -100 and -104.
This calculation accuracy can be improved in the same manner as the usual softmax calculation.

**Comment by masanorihirano** [comment]
I'll make a patch
